### ISSUE TITLE
Error when importing

### ISSUE THREAD

**Issue by DavidMartinezIsNutty** [FlailingFog/KK-Blender-Porter-Pack#286]
Please include the card that causes the issue!
It can really help with the debug process.
![Koikatu_M_20221010162720906_](https://user-images.githubusercontent.com/115497247/194927673-fd24763f-abd5-4dac-9ec1-944ca66d5885.png)

Python: Traceback (most recent call last):
  File "C:\Users\Miles\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\KK-Blender-Porter-Pack-V6-0-0\importing\importbuttons.py", line 88, in execute
    bpy.ops.kkb.separatebody('INVOKE_DEFAULT'),
  File "D:\Blender Foundation\Blender 3.00\3.0\scripts\modules\bpy\ops.py", line 130, in __call__
    ret = _op_call(self.idname_py(), C_dict, kw, C_exec, C_undo)
RuntimeError: Error: Traceback (most recent call last):
  File "C:\Users\Miles\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\KK-Blender-Porter-Pack-V6-0-0\importing\separatebody.py", line 648, in execute
    make_tear_and_gag_shapekeys()
  File "C:\Users\Miles\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\KK-Blender-Porter-Pack-V6-0-0\importing\separatebody.py", line 432, in make_tear_and_gag_shapekeys
    skey_driver = bpy.data.shape_keys[0].key_blocks['KK Eyes_gageye'].driver_add('value')
KeyError: 'bpy_prop_collection[key]: key "KK Eyes_gageye" not found'

location: <unknown location>:-1

I have tried troubleshooting the error I think the issue might lie with the exporter not having support for different head mods but I might be wrong I just want to make sure.

**Comment by Dolliex** [comment]
I'm having the same issue since today. Everything worked fine a day and a half ago when I first installed everything and I exported a model and everything went smooth. But today? Absolutely nothing works. I try to export onto blender and it keeps giving me the same errors as you. Uninstalling doesn't do anything. Using the master zip doesn't either. And it has nothing to do with head mods or anything because I tried it again with the model that DID work a day ago, and it's getting the same errors. Makes no sense and I'm at a loss of what to do.

**Comment by AkagiVoid** [comment]
> I'm having the same issue since today. Everything worked fine a day and a half ago when I first installed everything and I exported a model and everything went smooth. But today? Absolutely nothing works. I try to export onto blender and it keeps giving me the same errors as you. Uninstalling doesn't do anything. Using the master zip doesn't either. And it has nothing to do with head mods or anything because I tried it again with the model that DID work a day ago, and it's getting the same errors. Makes no sense and I'm at a loss of what to do.

The best U can do atm is to downgrade KKBP to 6.0.0 for now until the creator/Contributors fix things. X3
The 6.0.0 is only giving my avatar body texture sum weird blue pixelation error for the material node (and haven't tested the export model feature yet btw). >w<
I also started a thread with mar in depth of the error at #287 . ^w^

**Comment by Dolliex** [comment]
> > I'm having the same issue since today. Everything worked fine a day and a half ago when I first installed everything and I exported a model and everything went smooth. But today? Absolutely nothing works. I try to export onto blender and it keeps giving me the same errors as you. Uninstalling doesn't do anything. Using the master zip doesn't either. And it has nothing to do with head mods or anything because I tried it again with the model that DID work a day ago, and it's getting the same errors. Makes no sense and I'm at a loss of what to do.
> 
> The best U can do atm is to downgrade KKBP to 6.0.0 for now until the creator/Contributors fix things. X3 The 6.0.0 is only giving my avatar body texture sum weird blue pixelation error for the material node (and haven't tested the export model feature yet btw). >w< I also started a thread with mar in depth of the Importing Error at #287 . ^w^

I tried 6.0.0 and it didn't let me import the model at all either. Do I have to download the KKBP Exporter 3.90 for it to function?

**Comment by AkagiVoid** [comment]
> I tried 6.0.0 and it didn't let me import the model at all either. Do I have to download the KKBP Exporter 3.90 for it to function? I currently still have the 3.94 one.

First, let's break down sum pointers here.
First what Koikatsu are you using?
And what Blender Version are you using as well?

(I'm using Koikastu BR15, and Blender 3.3.1, with KKBP_Exporter_**V3.94** **AND** KK-Blender-Porter-Pack-**V6.0.0**.
The KKBP export for the Koikatsu game works fine, it's the KK-Blender-Porter-Pack-V6.1.0 that is broken atm. 
So when I downgraded it to KK-Blender-Porter-Pack-V6.0.0 everything is working fine except for a tiny problem where it keeps messing up in exports for MT_CT in the fbx wrong I'll post an image if it below).
![image](https://user-images.githubusercontent.com/48104310/195010459-36d9ce0e-d89c-482f-8850-da6235d34117.png)


And last note the KK-Blender-Porter-Pack just got updated to V6.1.1 btw (Gonna go smash test dat Plugin r8t now). :3

**Comment by Dolliex** [comment]
I installed Koikatsu Party, with the HF patch, KKBP Exporter 3.94 and KK Blender 6.1.1 and also got Blender 3.3 for it all. Everything worked fine and imported correctly into blender, and KKBP was actually doing its' job prepping up my model and all, but after I took a break from it for a day and a half, I came back and tried with 2 other models and they just fail at being imported onto blender. I even tried the first model that did work a couple days ago, and even that one was being errored out by the plugin suddenly. Today I tried blender 2.92, 3.0.0 and tried KK Blender 6.0.0 as well but nothing works for me. When I hit import model it takes 3 mins and gives me a white model and a big error from blender/plugin and then trying to prep the rig up or even bake materials are outright refused when I try to. This happens consistently on any model I try and any version of blender and KK Blender Exporter. I noticed the plugin was update a few hours ago so I gave it a try too but nope, same errors. So unsure of what to do really. The only difference between the day everything worked vs today nothing working, is the fact I restarted my computer so it doesn't make sense, ugh.
